### PR TITLE
Update http4s to 0.19

### PR DESCRIPTION
**Pull request by hgiasac**
PS: Before merge this branch, could you take a look at 0.16 PR #5 and 0.18 #6 branch? Thank so much!

Every new version of http4s has a big change, the current template code is out of dated. This PR update http4s to 0.19

However, It's better to create branches for each version, same as https://github.com/http4s/http4s.g8. I have a PR for version 0.18 and/or 0.16 too. Can you create 0.18 branch so I can create that PR? Thanks

### CODE REVIEW

File: src/main/g8/src/main/scala/$package__packaged$/main/Application.scala [hunk 1]
```diff
@@ -1,37 +1,33 @@
...
-    else
-      Left(results)
+      println("Http4s Server Started successfully")
+      components.httpServer
+        .stream
+        .compile
+        .drain
+        .as(ExitCode.Success)
+    } else {
+      println("Failed to start Http4s server")
```
Comment (jcranky) [inline src/main/g8/src/main/scala/$package__packaged$/main/Application.scala:27]: I would prefer to encapsulate the `println`s in `delay`s.

**Comment by jcranky** [comment]
Thanks. I had just a small comment, that noticed only after merging.

**Comment by hgiasac** [comment]
I am appreciated for you feedback. Would you mind change it for me? Thanks for advance.

**Comment by jcranky** [comment]
@hgiasac sure, I'll do that later :)

**Comment by jcranky** [comment]
@hgiasac so, just in case you are curious, this is what I meant: https://github.com/jcranky/grafter-http4s.g8/pull/7

**Comment by hgiasac** [comment]
@jcranky Thanks

**Pull request merged**